gutenberg.php

```diff
@@ -28,6 +28,25 @@
 function the_gutenberg_project() {
 	global $post_type_object;
 	?>
+	<noscript>
+		<div class="error" style="position:absolute;top:32px;z-index:40"><p>
+		<?php
+			// Using Gutenberg as Plugin
+			if ( is_plugin_active( 'gutenberg/gutenberg.php' ) ) {
+				$current_url = esc_url( $_SERVER['REQUEST_URI'] . "&classic-editor" );
```

Comment:
This (concatenating `"&classic-editor"`) will break if `REQUEST_URI` doesn't carry prior query arguments. Should use `add_query_arg` or equivalent.

Reply:
I've updated, though I'm not sure if that can happen on the editor page.

Reply:
> I've updated, though I'm not sure if that can happen on the editor page.

By default, Add New Post leads to `/wp-admin/post-new.php` and Add New Page leads to `/wp-admin/post-new.php?post_type=page`. Both situations are to be expected, so `add_query_arg` is definitely necessary.